assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -18,4 +18,9 @@ module "bootkube" {
 
   # Disable the self hosted kubelet.
   disable_self_hosted_kubelet = var.disable_self_hosted_kubelet
+
+  # Block access to AWS metadata for all pods.
+  #
+  # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
+  blocked_metadata_cidrs = ["169.254.169.254"]
```

Comment:
@invidian if it's a CIDR (as I see in the calico docs, if I'm not missing something) shouldn't we use `ip/32` format?  (this applies to all variable definitins without the `/32` in the PR).

Adding @iaguis as he might be more involved with calico, maybe?

Reply:
You're right. PR incoming.

Reply:
I think [calico will handle it](https://github.com/projectcalico/libcalico-go/blob/1b8bc835691e7e166a2878e482178c44fd4587bf/lib/net/ipnet.go#L40) but yes, you're right.

Reply:
Cool, thanks! :)

Reply:
Thanks!